### PR TITLE
Chore(readme): use https

### PR DESCRIPTION
**Pull request by imba-tjd**

### CODE REVIEW

**Comment by Cheesebaron** [comment]
Thanks. Can you change the target to master instead?

**Comment by imba-tjd** [comment]
Do I need to close this and open another PR?

File: global.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
+  "sdk": {
+    "version": "3.0.100"
```
Comment (onovotny) [inline global.json]: There's no need to pin a version, only makes things harder

File: azure-pipelines.yml [hunk 2]
```diff
@@ -24,6 +24,12 @@ variables:
   DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
 
 steps:
+- task: UseDotNet@2
+  displayName: 'Use .NET Core sdk'
+  inputs:
+    packageType: sdk
+    version: 3.0.100
```
Comment (onovotny) [inline azure-pipelines.yml]: I'd use '3.0.x' as the version as that ensures you always use the latest patch release (fixes/security updates)

**Pull request merged**